test/gitmojiCli.spec.js

```diff
@@ -81,28 +81,26 @@ describe('gitmoji module', () => {
   describe('commit', () => {
     it('should match for the commit snapshot with the given prompts', () => {
       config.setIssueFormat('github')
+      config.setSignedCommit(true)
       expect(gitmojiCli._commit(stubs.prompts)).toMatchSnapshot()
     })
 
     it('should match for the commit snapshot with the given prompts', () => {
       config.setIssueFormat('jira')
+      config.setSignedCommit(true)
       expect(gitmojiCli._commit(stubs.promptsJira)).toMatchSnapshot()
     })
+
+    it('should match for the commit snapshot with the given prompts', () => {
```

Comment:
We don't need to create this specific test, just `setSignedCommit` to false on the jira one 👍